ros_buildfarm/templates/devel/devel_job.xml.em

```diff
@@ -219,7 +222,8 @@ if pull_request:
         'echo "# BEGIN SECTION: Run Dockerfile - build and test"',
         ''
         'if [ ! -d "$HOME/.ccache" ]; then mkdir $HOME/.ccache; fi',
-        'docker run' +
+        '',
+        ('docker run --env=DISPLAY=:0.0 --env=QT_X11_NO_MITSHM=1 --volume=/tmp/.X11-unix:/tmp/.X11-unix:rw  --gpus all' if require_gpu_support else 'docker run') +
```

Comment:
Let's keep the common part - `docker run` - outside of the condition and only conditional inserr the gpu specific.

Reply:
agree b0d0736